Userland/DynamicLoader/main.cpp

```diff
@@ -79,33 +151,10 @@ NAKED void _start(int, char**, char**)
 #endif
 }
 
-static ErrorOr<int> open_executable(char const* path)
-{
-    int rc = open(path, O_RDONLY | O_EXEC);
-    if (rc < 0)
-        return Error::from_errno(errno);
-    int checked_fd = rc;
-    ArmedScopeGuard close_on_failure([checked_fd] {
-        close(checked_fd);
-    });
-
-    struct stat executable_stat = {};
-    rc = fstat(checked_fd, &executable_stat);
-    if (rc < 0)
-        return Error::from_errno(errno);
-    if (!S_ISREG(executable_stat.st_mode)) {
-        if (S_ISDIR(executable_stat.st_mode))
-            return Error::from_errno(EISDIR);
-        return Error::from_errno(EINVAL);
-    }
-
-    close_on_failure.disarm();
-    return checked_fd;
-}
-
 ALWAYS_INLINE static void optimizer_fence()
 {
-    asm("" ::: "memory");
+    asm("" ::
+            : "memory");
```

Comment:
I think this happened because of clang-format 16 (and Serenity uses clang-format 18 now)

Reply:
I have clang-format 17 on my machine (which is not updated yet until archlinux publishes an update).